code/modules/experisci/experiment/experiments.dm

```diff
@@ -0,0 +1,139 @@
+							/obj/item/slime_extract/gold)
+
+/datum/experiment/scanning/random/slime/hard
+	name = "Challenging Slime Survey"
+	description = "Another station has challenged your research team to collect several challenging slime cores, \
+	 				including a very valuable rainbow core. Are you up to the task?"
+	required_atoms = list(/obj/item/slime_extract/rainbow = 1)
+	possible_types = list(/obj/item/slime_extract/oil,
+							/obj/item/slime_extract/black,
+							/obj/item/slime_extract/lightpink,
+							/obj/item/slime_extract/adamantine)
+
+
+/datum/experiment/scanning/random/cytology/easy
+	name = "Basic Cytology Scanning Experiment"
+	description = "A scientist needs vermin to test on, use the cytology equipment to grow some of these simple critters!"
+	total_requirement = 3
+	max_requirement_per_type = 2
+	possible_types = list(/mob/living/simple_animal/hostile/cockroach, /datum/micro_organism/cell_line/mouse)
+
+/datum/experiment/scanning/random/cytology/medium
+	name = "Advanced Cytology Scanning Experiment"
+	description = "We need to see how the body functions from the earliest moments. Some cytology experiments will help us gain this understanding."
+	total_requirement = 3
+	max_requirement_per_type = 2
+	possible_types = list(/mob/living/simple_animal/hostile/carp, /mob/living/simple_animal/hostile/retaliate/poison/snake, /mob/living/simple_animal/pet/cat, /mob/living/simple_animal/pet/dog/corgi, /mob/living/simple_animal/cow, /mob/living/simple_animal/chicken)
+
+/datum/experiment/scanning/random/cytology/medium/one
+	name = "Advanced Cytology Scanning Experiment One"
+
+/datum/experiment/scanning/random/cytology/medium/two
+	name = "Advanced Cytology Scanning Experiment Two"
+
+/datum/experiment/scanning/random/janitor_trash
+	name = "Station Hygiene Inspection"
+	description = "To learn how to clean, we must first learn what it is to have filth. We need you to scan some filth around the station."
+	possible_types = list(/obj/effect/decal/cleanable/vomit,
+	/obj/effect/decal/cleanable/blood/splatter)
+	total_requirement = 3
+
+/datum/experiment/explosion/calibration
+	name = "Is This Thing On?"
+	description = "The engineers from last shift left a notice for us that the doppler array seemed to be malfunctioning. \
+					Could you check that it is still working? Any explosion will do!"
+	required_light = 1
+
+/datum/experiment/explosion/maxcap
+	name = "Mother of God"
+	description = "A recent outbreak of a blood-cult in a nearby sector necessitates the development of a large explosive. \
+					Create a large enough explosion to prove your bomb, we'll be watching."
+
+/datum/experiment/explosion/calibration
+	name = "Is This Thing On?"
+	description = "The engineers from last shift left a notice for us that the doppler array seemed to be malfunctioning. \
+					Could you check that it is still working? Any explosion will do!"
+	required_light = 1
+
+/datum/experiment/explosion/medium
+	name = "Is This Thing On?"
```

Comment:
I would recommend making the medium explosion tier have a different name in order to be not confusing as to how your (trash) bomb worked on one experiment but on the exact same experiment did not work.

Reply:
still WIP but yea ill change it